packages/fxa-auth-db-mysql/lib/db/schema/patch-097-098.sql

```diff
@@ -0,0 +1,194 @@
+--
+-- This migration introduces the following subscription-related data collections:
+--
+--  * Capabilities relevant to clients
+--  * Capabilities granted by products
+--  * Products enabled via subscription
+--  * Subscriptions enabled for accounts
+--
+
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CREATE TABLE IF NOT EXISTS subscriptionCapabilities (
+  id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  name VARCHAR(191) NOT NULL UNIQUE KEY,
+  url VARCHAR(256),
+  description VARCHAR(256)
+) ENGINE=InnoDB;
+
+CREATE TABLE IF NOT EXISTS subscriptionProducts (
+  id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  -- https://stripe.com/docs/api/service_products/object#service_product_object-id
+  subhubProductId VARCHAR(191) NOT NULL UNIQUE,
+  name VARCHAR(256),
+  url VARCHAR(256)
+) ENGINE=InnoDB;
+
+CREATE TABLE IF NOT EXISTS subscriptionProductCapabilities (
+  id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  productId INT UNSIGNED NOT NULL,
+  capabilityId INT UNSIGNED NOT NULL,
+  UNIQUE INDEX subscriptionProductCapabilitiesUnique(productId, capabilityId)
+) ENGINE=InnoDB;
+
+CREATE TABLE IF NOT EXISTS subscriptionClientCapabilities (
+  id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  -- fxa_oauth.clients.id
+  clientId BINARY(8) NOT NULL,
+  capabilityId INT UNSIGNED NOT NULL,
+  UNIQUE INDEX subscriptionClientCapabilitiesUnique(clientId, capabilityId)
+) ENGINE=InnoDB;
+
+CREATE TABLE IF NOT EXISTS accountSubscriptionItems (
+  id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  uid BINARY(16) NOT NULL,
+  productId INT UNSIGNED NOT NULL,
+  -- https://stripe.com/docs/api/customers/object#customer_object-id
+  subhubCustomerId VARCHAR(191),
+  -- https://stripe.com/docs/api/subscriptions/object#subscription_object-id
+  subhubSubscriptionId VARCHAR(191),
+  -- https://stripe.com/docs/api/subscription_items/object#subscription_item_object-id
+  subhubSubscriptionItemId VARCHAR(191),
+  createdAt BIGINT SIGNED NOT NULL,
+  FOREIGN KEY (uid) REFERENCES accounts(uid) ON DELETE CASCADE,
+  FOREIGN KEY (productId) REFERENCES subscriptionProducts(id) ON DELETE CASCADE,
```

Comment:
Sadly these tend to give us trouble with future db migrations, ref https://github.com/mozilla/fxa-auth-db-mysql/issues/485